test/box-luatest/lua_memory_test.lua

```diff
@@ -0,0 +1,68 @@
+local server = require('luatest.server')
+local t = require('luatest')
+local g = t.group('lua_memory setting test')
+--local treegen = require('test.treegen')
+
+local MB = 1024 * 1024
+local LUA_MIN_MEMORY = 256 * MB
+
+g.before_each(function()
+    g.server = server:new{
```

Comment:
Please use `()` for function or methods calls. Here and below.

Reply:
Done.